variants/jczn_2432s028r/pins_arduino.h

```diff
@@ -0,0 +1,85 @@
+
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+static const uint8_t TX = 1;
+static const uint8_t RX = 3;
+
+static const uint8_t D35 = 35;
+static const uint8_t D22 = 22;
+static const uint8_t D27 = 27;
+static const uint8_t D21 = 21;
+
+static const uint8_t A6  = 34;
+static const uint8_t A17 = 27;
+
+static const uint8_t T7 = 27;
+
+static const uint8_t SDA = 21;
+static const uint8_t SCL = 22;
+
+static const uint8_t SS   =  5;
+static const uint8_t MOSI = 23;
+static const uint8_t MISO = 19;
+static const uint8_t SCK  = 18;
+
+#define CYD_TP_IRQ  36
+#define CYD_TP_MOSI 32
+#define CYD_TP_MISO 39
+#define CYD_TP_CLK  25
+#define CYD_TP_CS   33
+#define CYD_TP_DIN  CYD_TP_MOSI
+#define CYD_TP_OUT  CYD_TP_MOSI
+#define CYD_TP_SPI_BUS  VSPI
+
+#define CYD_TFT_DC    2
+#define CYD_TFT_MISO 12
+#define CYD_TFT_MOSI 13
+#define CYD_TFT_SCK  14
+#define CYD_TFT_CS   15
+#define CYD_TFT_RS	 CYD_TFT_DC
+#define CYD_TFT_SDO	 CYD_TFT_MISO
+#define CYD_TFT_SDI	 CYD_TFT_MOSI
+#define CYD_TFT_SPI_BUS  HSPI
+
+#define CYD_TFT_WIDTH     320
+#define CYD_TFT_HEIGHT    240
+#define CYD_SCREEN_WIDTH  CYD_TFT_WIDTH
+#define CYD_SCREEN_HEIGHT CYD_TFT_HEIGHT
+
+#define CYD_TFT_BL        21
+#define CYD_TFT_BL_ENABLE ( ( pinMode( CYD_TFT_BL, OUTPUT ) ) )
```

Comment:
since you are calling a function, append `()` to the define: `CYD_TFT_BL_ENABLE()`. Same for the other such defines

Reply:
Great tip. Thanks.
Revisions complete.